### ISSUE TITLE
Add documentation how to use the trained network

### ISSUE THREAD

**Issue by MagicMuscleMan** [PumasAI/SimpleChains.jl#78]
All the examples in the documentation stop after the training. How to apply the trained model to new input data is missing or should be made more obvious.

**Comment by wardjm** [comment]
Did you figure it out?

**Comment by MagicMuscleMan** [comment]
> Did you figure it out?

Unfortunately not.

**Comment by wardjm** [comment]
Thanks! I will get it today out of necessity. I'll report back and maybe make a pull request.

**Comment by chriselrod** [comment]
`sc(x, p)` would give the loss, and
```julia
sc_noloss = SimpleChains.remove_loss(sc)
sc(x, p)
```
should give the predicted value.

**Comment by wardjm** [comment]
Thanks! Here's my entire script heavily based on the examples. It's REPL, so you'll need to slap some prints in there to run it as a script.

```
using MLDatasets
using SimpleChains

xtrain3, ytrain0 = MLDatasets.MNIST.traindata(Float32);
xtest3, ytest0 = MLDatasets.MNIST.testdata(Float32);

size(xtest3)
# (28, 28, 60000)
extrema(ytrain0) # digits, 0,...,9
# (0, 9)

xtrain4 = reshape(xtrain3, 28, 28, 1, :);
xtest4 = reshape(xtest3, 28, 28, 1, :);
ytrain1 = UInt32.(ytrain0 .+ 1);
ytest1 = UInt32.(ytest0 .+ 1);

lenet = SimpleChain(
  (static(28), static(28), static(1)),
  SimpleChains.Conv(SimpleChains.relu, (5, 5), 6),
  SimpleChains.MaxPool(2, 2),
  SimpleChains.Conv(SimpleChains.relu, (5, 5), 16),
  SimpleChains.MaxPool(2, 2),
  Flatten(3),
  TurboDense(SimpleChains.relu, 120),
  TurboDense(SimpleChains.relu, 84),
  TurboDense(identity, 10),
)

lenetloss = SimpleChains.add_loss(lenet, LogitCrossEntropyLoss(ytrain1));

p = SimpleChains.init_params(lenet);

G = SimpleChains.alloc_threaded_grad(lenetloss);

SimpleChains.train_batched!(G, p, lenetloss, xtrain4, SimpleChains.ADAM(3e-4), 10);

sc_noloss = SimpleChains.remove_loss(lenet)

# Do all:
pred = lenet(xtest4, p)

pred[:,1]

ytest[1]

# Do single:
m = xtest4[:,:,1,1]
m1 = reshape(m, 28,28,1,:)
pred1 = lenet(m1, p)

```

**Comment by mrazomej** [comment]
I agree that this is incredibly important! I spent the last 2 hours trying to figure out the simplest and most important thing: How to use the trained network. I should have come to the issues from the beginning since it was apparent I would be one of many with this question. But this has to be the first thing in the documentation.

**Comment by Vilin97** [comment]
Technically, there is an [example](https://pumasai.github.io/SimpleChains.jl/stable/#SimpleChains.SimpleChain) in the docs but it's very hidden away and I also did not find it before I knew exactly what to look for.

@chriselrod , would it make sense to add @wardjm 's example in the docs?
